.github/workflows/ci.yml

```diff
@@ -46,3 +46,25 @@ jobs:
         f3d_version: origin/master
         raytracing_label: ${{matrix.raytracing_label}}
         egl_label: ${{matrix.egl_label}}
+
+  python_wheels:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu, windows, macos]
+
+    runs-on: ${{matrix.os}}-latest
+
+    env:
+      DISPLAY: :0
+
+    steps:
+
+    - name: Checkout
```

Comment:
Any reason you are not putting this step in the action?

Reply:
you need the code of an action to use it, see the next step

Reply:
Yeah, obviously, thanks!